<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeutronMonitorDataAnalizer</class>
 <widget class="QWidget" name="NeutronMonitorDataAnalizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeutronMonitorDataAnalizer</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>781</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_layout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="select_file_layout">
      <item>
       <widget class="QLineEdit" name="neutron_monitor_data_file_path_input">
        <property name="placeholderText">
         <string>Write the path to the file with the neutron monitor data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="select_file_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="analize_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Analize neutron monitor data for the selected day</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="analysis_result_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Analysis result: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
